client/my-sites/site-settings/disconnect-site/disconnect-survey.jsx

```diff
@@ -39,38 +39,38 @@ class DisconnectSurvey extends PureComponent {
 	};
 
 	render() {
-		const { confirmHref, isPaidPlan, siteId, siteSlug, translate } = this.props;
+		const { confirmHref, siteId, translate } = this.props;
 		return (
 			<div className="disconnect-site__survey main">
 				<QuerySitePlans siteId={ siteId } />
 				<SettingsSectionHeader
-					title={ translate(
-						'Would you mind sharing why you want to disconnect %(siteName)s from WordPress.com?',
-						{
-							args: { siteName: siteSlug },
-						}
-					) }
+					title={ translate( 'Your feedback will help us improve the product.' ) }
 				/>
-				<CompactCard href={ `/settings/disconnect-site/too-difficult/${ siteSlug }` }>
-					{ translate( 'It was too hard to configure Jetpack.' ) }
-				</CompactCard>
-				<CompactCard href={ `/settings/disconnect-site/missing-feature/${ siteSlug }` }>
-					{ translate( 'A feature I need was missing.' ) }
-				</CompactCard>
-				{ isPaidPlan && (
-					<CompactCard href={ `/settings/disconnect-site/too-expensive/${ siteSlug }` }>
-						{ translate( 'This plan is too expensive.' ) }
-					</CompactCard>
-				) }
 				<CompactCard href={ confirmHref + '?reason=troubleshooting' }>
 					{ translate( "Troubleshooting — I'll be reconnecting afterwards." ) }
 				</CompactCard>
+				<CompactCard href={ confirmHref + '?reason=cannot-work' }>
+					{ translate( "I can't get it to work." ) }
+				</CompactCard>
+				<CompactCard href={ confirmHref + '?reason=slow' }>
+					{ translate( 'It slowed down my site.' ) }
+				</CompactCard>
+
+				<CompactCard href={ confirmHref + '?reason=buggy' }>
+					{ translate( "It's buggy." ) }
+				</CompactCard>
+				<CompactCard href={ confirmHref + '?reason=no-clarity' }>
+					{ translate( "I don't know what it does." ) }
+				</CompactCard>
+				<CompactCard href={ confirmHref + '?reason=delete' }>
+					{ translate( "I'm deleting or migrating my site." ) }
+				</CompactCard>
 				<CompactCard
 					className="disconnect-site__survey-other-option"
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Other', { context: 'Some other choice was made'} )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/survey-disconnect-refresh)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).